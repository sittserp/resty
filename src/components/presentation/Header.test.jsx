import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './Header';

describe('Header component', () => {
  afterEach(() => cleanup());
  it('renders Header', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
