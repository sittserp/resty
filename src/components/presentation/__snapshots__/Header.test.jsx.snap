// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header 1`] = `
<DocumentFragment>
  <header>
    <h1>
      RESTlify
    </h1>
  </header>
</DocumentFragment>
`;
